<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="msgBrowser">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="msgTextEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>340</y>
     <width>551</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="userTableWidget">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>0</y>
     <width>185</width>
     <height>421</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <column>
    <property name="text">
     <string>用户名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>IP地址</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>83</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QFontComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="currentFont">
    <font>
     <family>方正黑体_GBK</family>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="sizeComboBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>86</width>
     <height>32</height>
    </rect>
   </property>
   <property name="currentText">
    <string>12</string>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="userNumLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>420</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>在线用户：0人</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>420</y>
     <width>83</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_1">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_2">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>300</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>I</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>300</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>U</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_7">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>300</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QToolButton" name="sendTBtn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>300</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_9">
   <property name="geometry">
    <rect>
     <x>475</x>
     <y>300</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_10">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>300</y>
     <width>33</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
